dist/LICENSE

```diff
@@ -456,6 +456,7 @@ MIT licenses
     fill-range 7.0.1 MIT
     follow-redirects 1.15.2 MIT
     form-data 4.0.0 MIT
+    fsevents 2.3.2 MIT
```

Comment:
Where is it from? You've not introduced it in `go.mod` yet.

Reply:
This is I run make license-dep and it generate automatically

Reply:
Please remove your local `<banyandb_root_dir>/bin`. And rerun this command.

Reply:
The result of removing and rerunning make license-dep is the same.